### DISCUSSION TITLE
变量与历史记录 问题

### DISCUSSION THREAD

**Discussion by qiufuyu123** [Physics365/991CN-X-CW-Decompilation#21] [Q&A] [answered]
![image](https://github.com/Physics365/991CN-X-CW-Decompilation/assets/60765480/f8a0fa2b-bb38-402a-bc40-87a4bcd1d498)
能稍微细讲以下这部分怎么操作吗？没学会（笑哭）

**Comment by EwTE-H** [comment, marked as answer]
文档已修改，现在行了吗

**Comment by qiufuyu123** [comment]
ok明白了
